com.hack_computer.Clippy.Extension.json

```diff
@@ -0,0 +1,41 @@
+{
+    "app-id": "com.hack_computer.Clippy.Extension",
+    "branch": "stable",
+    "runtime": "com.hack_computer.Clubhouse",
+    "runtime-version": "stable",
+    "sdk": "org.gnome.Sdk//3.36",
+    "appstream-compose": false,
+    "separate-locales": false,
+    "build-extension": true,
+    "modules": [
+        {
+            "name": "clippy",
+            "buildsystem": "meson",
+            "config-opts": [
+                "-Dgtk-modules-path=/app/clippy/lib"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/endlessm/clippy.git",
+                    "branch": "master",
+                    "commit": "c3e78fa02d4193d617860f72fd603a05628c3602"
+                }
+            ]
+        },
+        {
+            "name": "appdata",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -Dm644 --target-directory=${FLATPAK_DEST}/share/appdata com.hack_computer.Clippy.Extension.appdata.xml",
+                "appstream-compose --basename=com.hack_computer.Clippy.Extension --prefix=${FLATPAK_DEST} --origin=flatpak com.hack_computer.Clippy.Extension"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "path": "data/com.hack_computer.Clippy.Extension.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Reply:
Thanks, I've updated that